src/emqx_misc.erl

```diff
@@ -529,6 +530,19 @@ redact(K, V) ->
 redact_v(V) when is_binary(V) -> <<?REDACT_VAL>>;
 redact_v(_V) -> ?REDACT_VAL.
 
+force_relative_filename_join([H | T]) ->
```

Comment:
If the purpose of this function is to deal with unintended leading `/`'s in the relative paths then why don't just use `filename:append` ?